platform/ios/src/MGLCameraChange.h

```diff
@@ -0,0 +1,35 @@
+#import "MGLFoundation.h"
+
+typedef NS_OPTIONS(NSUInteger, MGLCameraChangeReason)
```

Comment:
Is `MGLCameraChangeReason` a bitmask, rather than an ordinary enumeration, because a single gesture can lead to multiple of these reasons? Which combinations are possible? We should avoid a bitmask if possible, because less experienced programmers are much less likely to know how to perform bitwise operations.